bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/command/DebugCommand.java

```diff
@@ -425,8 +423,6 @@ private void sendDirectMessage(Console console, MessageType messageType, String[
                 console.println(msg.toString());
             } catch (FieldException | InvalidMessageTypeException | NumberFormatException e) {
                 console.println("Error while trying to create message.");
-            } catch (IOException e) {
```

Comment:
This exception is no longer needed because the data is always coming from the cache?

Reply:
Correct, `writeMessage` writes to a queue and not directly to the IO stream.

That function was handling `null` message it receives by throwing `IOException`. Since this was technically dead code, the `@Nullable` annotation was removed along with the code in question. This is a leftover from the legacy implementation.

https://github.com/openhab/openhab-addons/blob/248c4643bacb01129fea88dd63f81be57c7125c3/bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/transport/Port.java#L161-L164